private_gpt/server/chat/chat_service.py

```diff
@@ -108,6 +112,9 @@ def _chat_engine(
                 llm=self.llm_component.llm,  # Takes no effect at the moment
                 node_postprocessors=[
                     MetadataReplacementPostProcessor(target_metadata_key="window"),
+                    SimilarityPostprocessor(
+                        similarity_cutoff=settings().llm.similarity_value
```

Comment:
Settings should be injected in the ChatService constructor instead of called like `settings()`. It is explained in the `settings()` documentation.

Check, for example, `VectorStoreComponent` to see how to do it. 
```
@singleton
class VectorStoreComponent:
    settings: Settings
    vector_store: VectorStore

    @inject
    def __init__(self, settings: Settings) -> None: 
        self.settings = settings
```